README.md

```diff
@@ -11,7 +11,9 @@ In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
 def deps do
-  [{:ex_machina, "~> 0.4"}]
+  [
+    {:ex_machina, "~> 0.4", only: "test"}
+  ]
```

Comment:
I like this change, but I think that it should remain on one line for the README. To keep things a bit more concise